### PR TITLE
[AB#6554] New HCBRK (Kadaster) schema with auth

### PR DESCRIPTION
**Pull request by lbam**

### CODE REVIEW

File: datasets/haalcentraal/brk/dataset.json [hunk 1]
```diff
@@ -1,12 +1,11 @@
 {
   "type": "dataset",
-  "id": "hcbrk",
+  "id": "haalcentraal_brk",
   "title": "BRK van HaalCentraal",
   "status": "beschikbaar",
   "description": "Deze dataset maakt de HaalCentraal BRK API toegankelijk, bij gebruik van login credentials. De velden zijn gebaseerd op het schema van Haal-Centraal.",
```
Comment (jjmurre) [inline datasets/haalcentraal/brk/dataset.json:6]: Small nitpick: uniform spelling for "HaalCentraal".

File: datasets/haalcentraal/brk/dataset.json [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
   "version": "0.0.1",
   "crs": "EPSG:28992",
-  "auth": "BRK/RSN",
   "tables": [
     {
       "id": "kadastraalonroerendezaken",
```
Comment (jjmurre) [inline datasets/haalcentraal/brk/dataset.json:11]: Shouldn't there be an `identifier` for the tabel (as a (compound) PK)? Or, is that not neccesary for a virtual dataset?

File: datasets/haalcentraal/brk/dataset.json (hunk 1 above)
Comment (lbam) [reply datasets/haalcentraal/brk/dataset.json:6]: https://www.vngrealisatie.nl/producten/haal-centraal says "Haal Centraal", so changing to that.

File: datasets/haalcentraal/brk/dataset.json (hunk 2 above)
Comment (lbam) [reply datasets/haalcentraal/brk/dataset.json:11]: Added.

**Pull request closed without merging**